resources/lib/apihelper.py

```diff
@@ -162,7 +162,7 @@ def __map_seasons(self, program, seasons, episodes):
         season_items = []
         sort = 'label'
         ascending = True
-        content = 'files'
+        content = 'seasons'
```

Comment:
We did this IIRC to ensure folders were shown as folders, and not as files.
Which in fact is a general Kodi issue.

Reply:
If you don't set the mediatype on the ListItem, you get a folder icon in the Kodi skin.

Reply:
I know a lot of add-ons doing this wrong then :-D

Reply:
I don't understand the point of `mediatype` either.